pkg/credentialprovider/azure_credentials.go

```diff
@@ -232,24 +240,55 @@ func (a *acrProvider) getFromACR(ctx context.Context, loginServer string) (strin
 // parseACRLoginServerFromImage takes image as parameter and returns login server of it.
```

Comment:
Update the comment to describe what the function returns.

Reply:
done